dags/openshift_nightlies/config/install/hypershift/sdn.json

```diff
@@ -1,10 +1,9 @@
 {
-    "aws_profile": "",
-    "aws_access_key_id": "",
-    "aws_secret_access_key": "",
-    "rosa_environment": "staging",
-    "rosa_cli_version": "master",
+    "cluster_install_method": "osd",
+    "ocm_cli_version": "master",
     "ocm_environment": "stage",
+    "rosa_environment": "staging",
```

Comment:
Do we need rosa specific vars for the OSD scenario as well? I don't think we use these vars in OSD so we must be able to skip these as they are only consumed it the cluster_install_method is not osd right?

Reply:
yeah not required, will remove it.

Reply:
my bad, they are required to get the token for `ocm login` [here](https://github.com/cloud-bulldozer/airflow-kubernetes/pull/167/files#diff-2077ba877e816fdbbd1f6ebea9c103d42861a79b2549fb3e93e16ae708c93e01R63)